src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement/Application/UkrlpData/ProviderQueryResponse.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace SFA.DAS.RoatpCourseManagement.Application.UkrlpData
+{
+    [Serializable]
+    public class MatchingProviderRecords
+    {
+        [XmlElement]
+        public string UnitedKingdomProviderReferenceNumber { get; set; }
+
+        [XmlElement]
+        public string ProviderName { get; set; }
+
+        [XmlArray(ElementName = "ProviderContact")]
+        public List<ProviderContactStructure> ProviderContacts { get; set; }
+    }
+
+    [Serializable]
+    [XmlRoot(ElementName = "ProviderContact")]
+    public class ProviderContactStructure
+    {
+        [XmlElement]
+        public string ContactType { get; set; }
+
+        [XmlElement]
+        public ProviderContactAddress ContactAddress { get; set; }
+    }
+
+    [Serializable]
+    public class ProviderContactAddress
```

Comment:
ProviderAddress

Reply:
I'm leaving this as is, because when I change it, it creates complex mapping issues as it clashes with the Domain Entity 'ProviderAddress', and will create a code smell/complexity.